packages/patternlab/styleguide/source/_patterns/01-atoms/09-media/figure.twig

```diff
@@ -38,10 +38,10 @@
     {% endif %}
   </figure>
 {% if figure.skiplink.add %}
-  <div class="ma__figure__skip-link_target" id="{{ skiplinkId }}" >
-    <a tabindex="-1" aria-label="You skipped the {{ skiplinkText }}, {{ figure.title.text }}.">
-      You skipped the
-      {{ skiplinkText }}.
+  <div class="ma__figure__skip-link_target">
+    {# role and aria-disabled to ensure disabling the link functionality with a for AT. #}
+    <a id="{{ skiplinkId }}" role="link" aria-disabled="true" tabindex="-1">

```

Comment:
why not just `<a id="{{ skiplinkId }}"  tabindex="-1">`? 

This is still announced as a link in VO. I'm not seeing these attributes in the TOC jump link target

Reply:
The role and aria-disabled have been removed.